content/docs/started/workstation/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Workstation Install"
```

Comment:
Change "Install" (verb) to "Installation" (noun).

Reply:
Done